### ISSUE TITLE
There was an exception Number of processes must be at least 1

### ISSUE THREAD

**Issue by dlukeomalley** [returntocorp/bento#177]
This is the second time in the past week this issue has affected a user.

## Overview
If a user accidentally runs `bento init` in a directory without a `package.json` and no Python files, the following `.bento.yml` is produced:

```yaml
formatter:
  stylish: {}
tools: {}
```

When they then run `bento check`, the following message is displayed:

```shell
✘ There was an exception Number of processes must be at least 1
```

## Steps to Reproduce
1. Run `bento init` in the parent directory of a JavaScript only project
2. cd into the project, and run `bento check`
3. Also try running `bento init` again in the root directory of the project

## Possible Solution
1. `init` should clearly error if there are no supported languages found: https://github.com/returntocorp/bento/issues/170
2. `init` could run in the nested directory, and no search upward like `check` does
3. Error messages should have clear next steps the user can take, such as rerunning init or adding tools.

**Comment by nbrahms** [comment]
Fixed in 0.6:

- `init` will not create `.bento.yml` if the project is unsupported
- `check` will indicate if no tools are configured

**Comment by dlukeomalley** [comment]
Thank you @nbrahms! Are there corresponding acceptance/integration tests?
